python-sdk/docs/conf.py

```diff
@@ -51,6 +51,10 @@
     "show-inheritance",
     "show-module-summary",
 ]
+
+suppress_warnings = [
+    "autoapi.python_import_resolution",
```

Comment:
Yes, I should have added a comment -- this is because `airflow.dataset` is still not available for Airflow -- so Sphinx fails to resolve it -- which is why I am suppressing that error.